matchengine/main.py

```diff
@@ -71,7 +72,7 @@ def main(run_args):
     subp_p = subp.add_parser('load', help='Sets up your MongoDB for matching.')
     subp_p.add_argument('-t', dest='trial', default=None, help=param_trials_help)
     subp_p.add_argument('-c', dest='clinical', default=None, help=param_clinical_help)
-    subp_p.add_argument('-g', dest='extended_attributes', default=None, help=param_genomic_help)
+    subp_p.add_argument('-g', dest='genomic', default=None, help=param_genomic_help)
```

Comment:
While we can fix this, we should be discouraging using this "loading" functionality for anything other than trials, in favor of having users load data into MongoDB directly using e.g. an ETL process of some sort.